.github/workflows/ci.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           java-version: '17'
           cache: 'gradle'
       - name: Build
-        run: ./gradlew check -x test -x integration -x pmdTest -x pmdFunctionalTest -x pmdIntegrationTest
+        run: ./gradlew check -x integration
```

Comment:
Just checking this change is intentional - I _think_ the original exclusions were intended to avoid github CI failures for `ffmpeg` tests, as that executable isn't available in the github CI environment. Karen, John or Hemanta may know more.

Reply:
Yeah it is intentional. I believe the tests that use ffmpeg run as part of the integration suite. I think it's useful for Github actions to run all other tasks as it provides quicker feedback that Jenkins.